pkgs/development/tools/udis86/default.nix

```
@@ -0,0 +1,27 @@
+{ stdenv, fetchurl, autoreconfHook, python }:
+
+stdenv.mkDerivation rec {
+  pname = "udis86";
+  version = "1.7.2";
+
+  src = fetchurl {
+    url = https://github.com/vmt/udis86/archive/v1.7.2.tar.gz;
+    sha256 = "0yappb8dxwsq6jcqm4zq51iv6g53nzsx7czznp2l728n29z7ymj3";
+  };
+
+  nativeBuildInputs = [ autoreconfHook python ];
+
+  configureFlags = [
+    "--enable-shared"
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = http://udis86.sourceforge.net;
+    license = licenses.bsd2;
+    maintainers = with maintainers; [ timor ];
+    description = ''
+      Udis86 is an easy-to-use, minimalistic disassembler library (libudis86)
```

don't need to repeat ourselves :), and i think there's a preference to keep it on one line for the short description without a period at the end.
```suggestion
      Easy-to-use, minimalistic x86 disassembler library (libudis86)
```